fleet/api/http/openapi.yaml

```diff
@@ -742,6 +742,28 @@ components:
           type: string
           format: uuid
           description: Communication channel ID (UUIDv4), unique to this agent and created at agent creation
+        matching_groups:
+          type: object
+          description: Groups currently matching this agent
```

Comment:
Groups this Agent currently belongs to, according to matching agent and group tags